.gitignore

```diff
@@ -169,3 +169,4 @@ $RECYCLE.BIN/
 /build/DownloadSonarPlugin/ManualTestOutput
 /build/ProcessJarFiles/tmp
 /**/launchSettings.json
+/src/Rules/Embedded
```

Comment:
Ignore the generated files (there are thousands of them, and they're regenerated as part of the build).